lib/car.rb

```diff
@@ -0,0 +1,22 @@
+class Car
+  attr_reader :make, :model, :monthly_payment, :loan_length, :color
+  def initialize(name, monthly_payment, loan_length)
+    @name = name
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    @color
+
+    split = name.split(' ')
```

Comment:
Nice job getting this. This might be better to make into a method. So something like:
```
def name_split
   @name.split(' ')
end
```
Then in your initialize you could update `@make` and `@model` to:
```
@make = name_split.first
@model = name_split.last
```
Essentially what you have done in the initialize but in a method instead to clean up your initialize.